packages/react/README.md

```diff
@@ -66,6 +66,9 @@ Fetch data from the Solana blockchain with the gill hooks:
 - [`useTokenMint`](#get-token-mint-account) - get a decoded token's Mint account
 - [`useTokenAccount`](#get-token-account) - get the token account for a given mint and owner (or ATA)
 
+Send data to the Solana blockchain with the gill hooks:
```

Comment:
Would be great to add a usage example here to help developers understand how to use the hook:

```typescript
// Example usage
const { mutate, signature, isPending, isError } = useSendTransaction({
  wireTransaction: base64EncodedTransaction,
  config: { 
    skipPreflight: true,
    preflightCommitment: 'finalized'
  }
});

// Send the transaction
mutate();
```

Reply:
Also, the naming on mutate makes sense bu i think `send` would be a bit better